Scripts/bootstrap.sh

```diff
@@ -14,31 +23,32 @@ set -Eeuo pipefail
 
 trap "echo ; echo ❌ The Bootstrap script failed to finish without error. See the log above to debug. ; echo" ERR
 
-echo
-echo -e "👉 Install Mint if needed"
+puts "Install Mint if needed"
 # List installed Mint versions, if fails, install Mint
 brew list mint || brew install mint
 
 # Set bash to Strict Mode (http://redsymbol.net/articles/unofficial-bash-strict-mode/)
 set -euo pipefail
 
-echo
-echo -e "👉 Bootstrap Mint dependencies"
+puts "Bootstrap Mint dependencies"
```

Comment:
@alteral will this persist the `👉 ` emoji? :)

Reply:
Of course 🙂